ragna/assistants/_api.py

```diff
@@ -11,12 +11,10 @@
 from ragna.core import Assistant, EnvVarRequirement, RagnaException, Requirement, Source
 
 
-class ApiAssistant(Assistant):
-    _API_KEY_ENV_VAR: str
-
+class _ApiAssistant(Assistant):
     @classmethod
     def requirements(cls) -> list[Requirement]:
-        return [EnvVarRequirement(cls._API_KEY_ENV_VAR), *cls._extra_requirements()]
+        return []
```

Comment:
This is the default return if we don't override the method. Meaning, we can just leave it out here.